autogen/main/variables.tf.tmpl

```diff
@@ -402,24 +402,26 @@ variable "enable_private_endpoint" {
 variable "enable_private_nodes" {
   type        = bool
   description = "(Beta) Whether nodes have internal IP addresses only"
-  default     = false
+  default     = true
```

Comment:
@ericyz This seems unrelated?

Reply:
Sorry, mixed my local branch to this change, removing it now.